README.md

```diff
@@ -442,14 +449,18 @@ app.addActions({votes: voteActions})
 
 </details>
 
-### `app.addRoute(path, page)`
+### `app.addRoute(path, page[, subroutes])`
```

Comment:
Why is the comma added within the Array?

Reply:
this is the syntax used when a parameter is optional. It's not the universal standard, but I've seen it used in man pages. I'm open to other suggestions though.